packet-solutions/xdp_prog_kern_02.c

```diff
@@ -14,6 +14,15 @@
 #include "../common/xdp_stats_kern_user.h"
 #include "../common/xdp_stats_kern.h"
 
+/*
+static __always_inline __u16 csum_fold_helper(__u32 csum)
+{
+        __u32 sum;
+        sum = (csum >> 16) + (csum & 0xffff);
+        sum += (sum >> 16);
+        return ~sum;
+}
+*/
```

Comment:
The call to this below is there for illustrative purposes only, so I think having this here commented-out is more confusing than useful. So let's just get rid of it :)

Reply:
@tohojo Done.

Reply:
Great, thanks! :)